### PR TITLE
[SPARK-22268][BUILD] Fix lint-java

### PR DESCRIPTION
**Pull request by ash211**
## What changes were proposed in this pull request?

Fix java style issues

## How was this patch tested?

Run `./dev/lint-java` locally since it's not run on Jenkins

### CODE REVIEW

**Review by HyukjinKwon — APPROVED** [review]

File: core/src/test/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorterSuite.java [hunk 1]
```diff
@@ -530,9 +531,11 @@ public void testOOMDuringSpill() throws Exception {
     // we expect an OutOfMemoryError here, anything else (i.e the original NPE is a failure)
     catch (OutOfMemoryError oom){
       String oomStackTrace = Utils.exceptionString(oom);
-      assertThat("expected OutOfMemoryError in org.apache.spark.util.collection.unsafe.sort.UnsafeInMemorySorter.reset",
+      assertThat("expected OutOfMemoryError in " +
+              "org.apache.spark.util.collection.unsafe.sort.UnsafeInMemorySorter.reset",
               oomStackTrace,
-              Matchers.containsString("org.apache.spark.util.collection.unsafe.sort.UnsafeInMemorySorter.reset"));
+              Matchers.containsString(
+                  "org.apache.spark.util.collection.unsafe.sort.UnsafeInMemorySorter.reset"));
```
Comment (HyukjinKwon) [inline core/src/test/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorterSuite.java]: Wait .. @ash211, mind fixing these to double-spaced indentation?

**Comment by srowen** [comment]
Ping @ash211

**Comment by ash211** [comment]
Updated

**Review by srowen — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**